### ISSUE TITLE
Question about how to select domain labels

### ISSUE THREAD

**Issue by Freak0728** [daoyuan98/MSDA#3]
Hi, I am lucky to see your code, it helps me a lot, but one thing I can't understand is how to choose domain labels when calculating domain loss. I would appreciate it if you could answer this question.

**Comment by daoyuan98** [comment]
Hi, thank you for your interests in the code. The domain loss is actually calculated by a binary classification loss. Suppose there are source domains, {S1, S2, ..., Sk}, we calculate k domain losses by differentiating each source and the target domain T. For each source-target  pair {Si, T}, the domain label is either 0 or 1.

**Comment by Freak0728** [comment]
> 您好，感谢您对代码的兴趣。域损失实际上是通过二元分类损失计算的。假设有源域 {S1, S2, ..., Sk}，我们通过区分每个源域和目标域 T 来计算 k 个域损失。 对于每个源-目标对 {Si, T}，域标签是0 或 1。

**Comment by Freak0728** [comment]
Thank you for your reply, I saw your code address from the issue reply under MDAN code.
I don't know if you have seen the code of the article "MDAN: Multiple Source Domain Adaptation with Adversarial Learning", the code address is as follows: https://github.com/hanzhaoml/MDAN. 
In its Amazon sentiment analysis experiment, when calculating the domain loss, the source label was set to all 1 and the target label was set to all 0. I just don't understand the principle of this.
The code is as follows：
domain_losses = torch.stack([F.nll_loss(sdomains[j], slabels) +
                                           F.nll_loss(tdomains[j], tlabels) for j in range(num_domains)])

**Comment by daoyuan98** [comment]
Hi, I personally think this should not influence the results as long as the domain label is consistent across all source-target pairs, i.e. always set source=1&target=0 or source=0&target=1.

**Comment by Freak0728** [comment]
Okay, thanks very much, I understand it roughly!
